addons/lunch/wizard/lunch_cancel.py

```diff
@@ -1,29 +1,14 @@
 # -*- encoding: utf-8 -*-
-##############################################################################
-#
-#    OpenERP, Open Source Management Solution
-#    Copyright (C) 2004-2012 Tiny SPRL (<http://tiny.be>).
-#
-#    This program is free software: you can redistribute it and/or modify
-#    it under the terms of the GNU Affero General Public License as
-#    published by the Free Software Foundation, either version 3 of the
-#    License, or (at your option) any later version.
-#
-#    This program is distributed in the hope that it will be useful,
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
-from openerp.osv import fields, osv
 
-class lunch_cancel(osv.Model):
+from openerp import models, api
+
+
+class lunch_cancel(models.TransientModel):
```

Comment:
@tde-banana-odoo  : wizard is only used to ask confirmation `Are you sure you want to cancel these meals?` , and there are two other wizard for `order` and `validation` , we can remove it ?